src/GNDStk/utility.hpp

```diff
@@ -125,6 +126,13 @@ inline std::string diagnostic(
 // Shortcuts to njoy::Log functions
 // -----------------------------------------------------------------------------
 
+// Convenient, namespace-scope booleans for fine-tuning diagnostic output
+inline bool notes     = true; // general info messages
+inline bool debugging = true; // debug calls
+inline bool warnings  = true; // warnings
+inline bool context   = true; // any of our context-printing info messages
```

Comment:
Nice, I can now switch the warnings off for testing when I write small GNDS snippets

Reply:
I thought about writing "debugs" and "contexts" in order to have consistent language, although "debugs" doesn't come across as being a legitimate plural *noun*. What do you think?